app/labor/hex_comparer.rb

```diff
@@ -1,49 +1,49 @@
 class HexComparer
+  RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
+  ACCENT_MODIFIERS = [1.14, 1.08, 1.06, 0.96, 0.9, 0.8, 0.7, 0.6].freeze
+
   def initialize(hexes, amount = 1)
-    @hexes = hexes
+    @hexes = hexes.sort
```

Comment:
We never really use the `order` method from the outside, so inlining the sorting and removing the method seemed sensible.